README.md

```diff
@@ -6,11 +6,34 @@
 
 ## Overview
 
-AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. Originally [acs-engine](https://github.com/Azure/acs-engine), the project has moved here (1) to evolve without harming its legacy non-k8s bits that other non-k8s Azure upstream projects depend upon, and (2) to better reflect its affinity with [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
+AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. AKS-Engine provides convenient tooling to quickly bootstrap clusters, and is the vm provisioning implementation for [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
```

Comment:
`vm` ->  `VM`

aks-engine does more than that for AKS (presently some upgrade + scale implementation, provisioning of other resources such as vnets, templates, etc.) so I don't think that's an accurate description of reality.

Reply:
What's a better way to express in a single clause a simple summary of the abstraction layer that aks-engine fulfills for AKS?

Reply:
`implements cluster provisioning and lifecycle operations for [AKS]()...`?